src/api/test/functional/source_controller_test.rb

```diff
@@ -1868,6 +1867,17 @@ def test_remove_and_undelete_operations
 
     # delete entire project
     prepare_request_with_user 'fredlibs', 'buildservice'
+    get '/source/kde4/_meta'
+    assert_response :success
+    meta_before_delete = @response.body
+    # kind of stupid, but do_change_project_meta_test modifies backend data and does not clean up
+    put '/source/kde4/_meta', meta_before_delete
+    assert_response :success
```

Comment:
I have to admit that I don't understand why this is needed / why this should have any effect.

But I trust you on that one